onto-viewer-web-app/src/test/java/configuration/ConfigLoaderTest.java

```diff
@@ -0,0 +1,189 @@
+package configuration;
```

Comment:
Change package to the correct one.

Reply:
Moreover, as you are testing features from the `onto-viewer-config-loader`tests should be placed in that module.